### ISSUE TITLE
v5.3.2 always errors with "RxJS could not find any global context (window, self, global)"

### ISSUE THREAD

**Issue by felixfbecker** [ReactiveX/rxjs#2598]
See https://github.com/sourcegraph/javascript-typescript-langserver/issues/246
```
Error: RxJS could not find any global context (window, self, global)
    at /home/travis/build/sourcegraph/javascript-typescript-langserver/node_modules/@reactivex/rxjs/dist/cjs/src/util/root.ts:26:9
```

**Comment by jayphelps** [comment]
Thanks @felixfbecker! Saw this too and know the root cause. I'm told a fix is in the works ASAP.

**Comment by jayphelps** [comment]
A fix is being tested here #2599 (seems to work)

**Comment by jayphelps** [comment]
Fix is out. Install `5.3.3` 🎉
